Implemented harvest error page

peterVorman opened the pull request:
Add option to display harvest errors on harvest search page

peterVorman committed:
implemented harvest error page

peterVorman committed:
fixed flake

jguo144 commented on ckanext/harvest/templates/snippets/show_errors_button.html:
```diff
@@ -0,0 +1,9 @@
+{% set authorized_user = h.check_access('harvest_source_create') %}
+
+{% if authorized_user %}
+  <a href="{{ h.current_url() + '?show_errors=true' if '?' not in h.current_url() else h.current_url() + '&show_errors=true'}}" class="btn btn-small">
```
It would be neater to move the link logic to a if conditional block.

jguo144 committed:
Improve show errors button

-Fix facet pill removal
-Add 'Hide Harvest Errors' button to hide harvest errors

jguo144 committed:
Add class to harvest errors button

jguo144 committed:
Refactor harvest error templates

jguo144 committed:
Fix harvest error link when not logged in

jguo144 committed:
Update indentation

jguo144 reviewed (approved):
LGTM
